### PR TITLE
Core working group notes, December 9th

### PR DESCRIPTION
**Pull request by brghena**
### Pull Request Overview

Notes from the Core working group call on December 9th, 2022.

### Testing Strategy

This pull request was tested by skimming it a bit.

### TODO or Help Wanted

Should be good to go.

### Documentation Updated

- [X] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [N/A] Ran `make prepush`.

### CODE REVIEW

**Review by hudson-ayers — APPROVED** [review]
Thanks! Just reading my comments, I already see several places where your paraphrasing of my comment is much clearer than what I actually said

File: doc/wg/core/notes/core-notes-2022-12-09.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+
+# Update on Licensing PRs
+ * CI PR - https://github.com/tock/tock/pull/3345
+ * Hudson: Johnathan made a CI PR for licensing, but it's blocked on more discussion on the original PR
+ * Original PR - https://github.com/tock/tock/pull/3318
+ * Pat: I'll pick that up soon, sorry
+ * Johnathan: Is there a reason we do the two lines (copyright and license) in the order we picked? I thought the other order was more common
+ * Pat: I thought the order mattered for some reason. It was some implicit top-down thing. It felt arbitrary to me and I don't feel strongly about it.
+ * Johnathan: The boilerplate for both MIT and Apache but the copyright first.
```
Comment (jrvanwhy) [inline doc/wg/core/notes/core-notes-2022-12-09.md:121]: `but` -> `put`

**Review by jrvanwhy — DISMISSED** [review]

File: doc/wg/core/notes/core-notes-2022-12-09.md (hunk 1 above)
Comment (brghena) [reply doc/wg/core/notes/core-notes-2022-12-09.md:121]:
```suggestion
 * Johnathan: The boilerplate for both MIT and Apache put the copyright first.
```

**Review by jrvanwhy — APPROVED** [review]

**Review by lschuermann — APPROVED** [review]
bors r+

**Pull request merged**